server/integration_tests/test_data/demo_feb2023/query_4/chart_config.json

```diff
@@ -71,39 +71,39 @@
         ],
         "statVarSpec": {
           "Count_Worker_NAICSAccommodationFoodServices_multiple_place_bar_block": {
-            "name": "Accommodation and Food Services Industry",
+            "name": "Accomodation/Food",
```

Comment:
This is vague. I think it's important to have "Industry" here

Reply:
Added